instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './PostContainer.css';
+import CommentSection from '../CommentSection/CommentSection'
+
```

Comment:
This component can also include more information about each post such as username, thumbnail image, and more.